resources/styles/components/student-dashboard/dashboard.less

```diff
@@ -43,11 +43,11 @@
   }
 
   [data-category=physics] .view-reference-guide {
-    .tutor-background-image("physics-book-thumbnail.png");
+    .tutor-background-image("courses/physics-book-thumbnail.png");
     background-size: auto 80px;
   }
   [data-category=biology] .view-reference-guide {
-    .tutor-background-image("biology-book-thumbnail.png");
+    .tutor-background-image("courses/biology-book-thumbnail.png");
     background-size: auto 80px;
   }
```

Comment:
All url() references in the stylesheets need to point to the location of the assets on-disk location.  Webpack examines the files and will either embed them into the css, or copy them all to the publicDir config (set to dist).